src/bootstrap/bootstrap.py

```diff
@@ -463,18 +469,27 @@ def downloading_llvm(self):
         return opt == "true" \
             or (opt == "if-available" and self.build == "x86_64-unknown-linux-gnu")
 
-    def _download_stage0_helper(self, filename, pattern, tarball_suffix, date=None):
+    def _download_stage0_helper(
```

Comment:
nit: let's call this download artifact or download bootstrap helper, stage0 permitting stage1 is confusing.

Reply:
I called it `download_component_helper`, the same way rustup calls these components.